### ISSUE TITLE
callback not declared in scope

### ISSUE THREAD

**Issue by RGpacmob** [openhomeautomation/adafruit-io-dashboard#2]
I don't know if this is an issue with the code or with my execution.  I copied the code into arduino and changed appropriate inputs (wifi network, aio key, etc), but when I verify the code or attempt to upload it, I get the following error: 
Arduino: 1.6.7 (Mac OS X), Board: "Arduino/Genuino Uno"

RG_IOTtest:41: error: 'callback' was not declared in this scope
 PubSubClient mqttclient("io.adafruit.com", 1883, callback, client);
                                                  ^
I am extremely new to arduino and c, so I don't know if this is and issue with the code or something that I am overlooking.

Thanks for Posting this project... It is exactly what I wanted to try out.

**Comment by marcoschwartz** [comment]
That's now fixed :)

**Comment by RGpacmob** [comment]
Marco, thanks for looking at this.
That didn't seem to fix it. When I verify/compile the code, I get a "
'callback' not declared in this scope error".
I have tried looking around a bit, but the combination of me being new to
arduino and adafruit io being new, there isn't too much that I can compare
to and make any sense of.
I followed some of your other projects and was able to get cc3000 and
sensors working right, just not able to compile this for adafruit io.
Any pointers, suggestions you have would be great.

Thanks.

rg
